src/controls/FormBuilder.ts

```diff
@@ -95,6 +96,7 @@ export class FormBuilder {
         if (type === 'choice') return new W.Widget_choice(this, this.schema, input, serial)
         if (type === 'choices') return new W.Widget_choices(this, this.schema, input, serial)
         if (type === 'markdown') return new W.Widget_markdown(this, this.schema, input, serial)
+        if (type === 'custom') return new W.Widget_custom(this, this.schema, input, serial)
         console.log(`🔴 unknown type ${type}`)
         exhaust(type)
```

Comment:
It seems the generic types are not working here

Reply:
I would say it's acceptable there, because this function is an internal plumbing thing